tyk-headless/README.md

```diff
@@ -6,4 +6,6 @@ Provides a Tyk Installation for Community Edition Gateways
 [![Install Tyk on Kubernetes](https://img.youtube.com/vi/mkyl38sBAF0/0.jpg)](https://www.youtube.com/watch?v=mkyl38sBAF0)
 
 ## Installation
-	helm install tyk-ce ./tyk-headless -n tyk
+	helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+	helm repo update
+	helm install tyk-ce tyk-helm/tyk-headless -n tyk
```

Comment:
I find it useful to add `--version` option. Also, it'd be helpful once we start releasing often.

Reply:
That's done now.